pkg/minikube/node/start.go

```diff
@@ -977,3 +977,8 @@ To see benchmarks checkout https://minikube.sigs.k8s.io/docs/benchmarks/cpuusage
 func ValidWindowsOSVersions() map[string]bool {
 	return map[string]bool{"2019": true, "2022": true}
 }
+
+// ValidOS lists the supported OSes
+func ValidOS() map[string]bool {
+	return map[string]bool{"linux": true, "windows": true, "darwin": true}
```

Comment:
```suggestion
func ValidOS() []string {
	return []string{"linux", "windows"}
```
I go back on my previous statement, to make it easier to use this in the flag lets go back to `[]string`